wordpress-mobile/WordPress-Android
# Issue/16150 blogging prompt card scaffolding

#16150 part 1 of 2

This PR adds a part of scaffolding and UI for Blogging Prompt card.

Card layout is still WIP.

[![Image from Gyazo](https://i.gyazo.com/24510d27453820b9f79c4d897d33c3cc.png)](https://gyazo.com/24510d27453820b9f79c4d897d33c3cc)

To test:
- Using wasabi version build variant, navigate to App Settings -> Debug settings and enable `blogging_prompts_remote_field` option.
- Navigate back to My Site and confirm that he Blogging Prompt card is visible.
- Turn off `blogging_prompts_remote_field` flag
- Navigate back to My Site and confirm that he Blogging Prompt card is  NOT visible.
- Install vanilla version of the and confirm that he Blogging Prompt card is NOT visible. 

## Regression Notes
1. Potential unintended areas of impact
- Blogging Prompt card is visible when FF is off.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Automated and manual testing.

3. What automated tests I added (or what prevented me from doing so)
- FF and dashboard card builder tests.

PR submission checklist:
